src/dex/verified/codetest.js

```diff
@@ -0,0 +1,16 @@
+// const {
```

Comment:
@Mohzcrea8me what is this file ? Seems like some home work you did

Reply:
I added it to gitignore, it's what I use for manual testing the calculations will be removed on next commit